### PR TITLE
Add benchmark app to openvino-dev package

### PR DESCRIPTION
**Pull request by slyubimt**
### Details:
 - *Add openvino.tools.benchmark  package and entry point*. 
 - *Configuration & Tests*: [connected MR](https://gitlab-icv.inn.intel.com/inference-engine/product-configs/merge_requests/1577).

### Tickets:
 - *52769*

### CODE REVIEW

File: tools/setup.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from setuptools import setup, find_packages
+
+with open("README.md", "r", encoding="utf-8") as f:
+    long_description = f.read()
+
+with open('benchmark/requirements.txt') as f:
+    required = f.read().splitlines()
+
+NAMESPACE = 'openvino.tools'
+packages = find_packages()
```
Comment (generalova-kate) [inline tools/setup.py:21]: please rename this variable to tools_packages to mitigate misunderstanding in the #43 line

File: tools/setup.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    description="OpenVINO™ Python* tools package",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    entry_points={
+        'console_scripts': [
+            'benchmark_app = openvino.tools.benchmark.main:main'],
+    },
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
```
Comment (generalova-kate) [inline tools/setup.py:38]: use Proprietary - Intel  license like for other packages

File: tools/setup.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+$ python setup.py sdist bdist_wheel
+"""
+
+from setuptools import setup, find_packages
+
+with open("README.md", "r", encoding="utf-8") as f:
+    long_description = f.read()
+
+with open('benchmark/requirements.txt') as f:
```
Comment (generalova-kate) [inline tools/setup.py:17]: how about windows paths with '\'

**Review by akuporos — APPROVED** [review]

File: tools/setup.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Use this script to create a wheel with OpenVINO™ Python* tools:
+
+$ python setup.py sdist bdist_wheel
+"""
```
Comment (generalova-kate) [inline tools/setup.py:10]: add notes that it works only with openvino pkg or add it as dependency

File: tools/setup.py (hunk 3 above)
Comment (slyubimt) [reply tools/setup.py:16]: This works on Windows, since path-like object is expected.

File: tools/setup.py (hunk 4 above)
Comment (slyubimt) [reply tools/setup.py:10]: Added to the install_requires list

File: tools/setup.py (hunk 2 above)
Comment (slyubimt) [reply tools/setup.py]: Corrected to:  Apache Software License

**Review by generalova-kate — APPROVED** [review]

**Review by azhogov — APPROVED** [review]

**Pull request merged**